airbyte-integrations/connectors/source-kobotoolbox/source_kobotoolbox/source.py

```diff
@@ -0,0 +1,189 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import json
+import re
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+from urllib.parse import parse_qsl, urlparse
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import IncrementalMixin, Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+stream_json_schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": True,
+    "properties": {
+        "_id": {
+            "type": [
+                "number",
+                "null",
+            ]
+        },
+        "_submission_time": {"type": ["string", "null"]},
+    },
+}
```

Comment:
Are you hard coding the schema to only ingest those two properties? Why no don't map the stream schemas?

Reply:
Did you try running the connector in the Airbyte UI is it capturing all fields from the schemas you have?

Reply:
The schema is different for each stream. We are not using normalization for this connector.